content/docs/user-guide/programming/memory/allocators.md

```diff
@@ -37,8 +35,8 @@ The following diagram illustrates the hierarchy of AZ memory allocators.
 To apply an allocator to your class, use the `AZ_CLASS_ALLOCATOR` macro in your class or directly call `AZ::AllocatorInstance`<*some\_allocator*>.
 
 AZCore relies on `AZ_CLASS_ALLOCATOR` to specify the default allocator for the class or on explicit `azcreate` and `azdestroy` calls that specify the allocator in their signature.
-+ If your class does not implement `AZ_CLASS_ALLOCATOR` and you call `new` or `delete`, `new` or `delete` calls use the global `operator new` or `operator delete`.
-+ If your class does not implement `AZ_CLASS_ALLOCATOR` and you call `aznew`, you must implement a `new` operator that uses the `aznew` call signature.
++ If your class does not implement `AZ_CLASS_ALLOCATOR` and you call `new` or `delete`, `new` or `delete` calls, the calls will use the global `operator new` or `operator delete`.
```

Comment:
new or delete is repeated?
```suggestion
+ If your class does not implement `AZ_CLASS_ALLOCATOR, calls to `new` or `delete` will use the global `operator new` or `operator delete`.
```

Reply:
Well after the command it was meant to say If your class does not implement `AZ_CLASS_ALLOCATOR` and you call `new` or `delete`, **then** the `new` or `delete` calls will use the global `operator new` and `operator delete` respectively.

But I like your rewrite.